message/write_message.go

```diff
@@ -86,6 +86,14 @@ func (m WriteInserts) Exists(tableName string) bool {
 	})
 }
 
+func (m WriteInserts) GetRecords() []arrow.Record {
```

Comment:
Could even add `WriteMigrateTables.GetTables()` but the migrate flag throws us. Could return two lists but it wouldn't look good.